project.clj

```diff
@@ -7,7 +7,7 @@
                  [duct/core "0.8.0"]
                  [duct/database.sql "0.1.0"]
                  [duct/logger "0.3.0"]
-                 [hikari-cp "2.13.0"]
+                 [hikari-cp "3.0.1"]
```

Comment:
If hikari-cp 3.x targets Clojure 1.11, the Clojure dependency for this library should be updated as well.